### DISCUSSION TITLE
Contig coverage after rnaSPAdes

### DISCUSSION THREAD

**Discussion by poecile-palustris** [ablab/spades#938] [Q&A] [answered]
Good morning,

I received the transcripts.fasta file, where k-mer coverage (here 11.9819) is specified (according to the manual):

`>NODE_97_length_6237_cov_11.9819_g8_i2`

Please tell me where can I find contig coverage? 

Thank you in advance!

Best regards,
Poecile

**Comment by asl** [comment, marked as answer]
Hello

This is exactly the contig coverage

**Reply by poecile-palustris** [reply]
Anton, right, thanks your for help!
